passport_expiration/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013 Savoir-faire Linux
+# Copyright 2021 Camptocamp SA
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Passport Expiration",
+    "version": "15.0.0.1.0",
```

Comment:
```suggestion
    "version": "15.0.1.0.0",
```

I have a little comment

Code review, ok :+1: